simpleprophet/setup.py

```diff
@@ -0,0 +1,63 @@
+"""Mozilla simpleprophet forecast framework."""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+# Arguments marked as "Required" below must be included for upload to PyPI.
+# Fields marked as "Optional" may be commented out.
+
+setup(
+    name='simpleprophet',
+    # use_scm_version={"root": "..", "relative_to": __file__},
+    # setup_requires=['setuptools_scm'],
```

Comment:
I thought this should use setuptools-scm?

Reply:
Thank you. I removed this for testing. Will revert.